models/staging/synthea/_synthea__models.yml

```diff
@@ -26,8 +26,6 @@ models:
       - name: patient_suffix
         data_type: text
         description: "Name suffix, such as PhD, MD, JD, etc."
-      - name: patient_maiden_name
```

Comment:
ah i think this should actually be `maiden_name`

Reply:
good catch!